### ISSUE TITLE
missing 1 required positional argument: 'mainArtists'

### ISSUE THREAD

**Issue by bamtan** [redsudo/RedSea#8]
I'm getting the following error when attempting to download the following: https://listen.tidal.com/album/73295425

```<<< Downloading album: 9 track(s) in total >>>
=== Downloading track ID 73295426 ===
        Track: 01
        Title: Intro
        Artist: Big Kahuna OG & Graymatter
        Album: Scooby Snax
        ----
Traceback (most recent call last):
  File "./redsea.py", line 143, in <module>
    main()
  File "./redsea.py", line 127, in main
    media_info)
  File "/mnt/c/Users/bamtan/Documents/redsea/redsea/mediadownloader.py", line 142, in download_media
    **self._normalise_info(track_info, album_info, True)))
  File "/mnt/c/Users/bamtan/Documents/redsea/redsea/mediadownloader.py", line 74, in _normalise_info
    for a in album_info['artists']
TypeError: get_artist_format() missing 1 required positional argument: 'mainArtists'
```

**Comment by redsudo** [comment]
This should now be fixed in the develop branch and will be released officially with version 1.1

If you can, pull develop and test/let me know how it goes

**Comment by bamtan** [comment]
Works perfectly now, am able to download the mentioned album! Thank you Sudo.
